### ISSUE TITLE
Fatal error generated on disabled single line text field on submission

### ISSUE THREAD

**Issue by roland-d** [verbb/formie#1282]
### Describe the bug

When submitting a page in a multi-page form I am greeted by the error

```
The request timed out.
Couldn’t save submission due to errors.
```

Looking at the console log, I have this JSON response (filepaths redacted):
```json
{
  "name": "Exception",
  "message": "Calling unknown method: verbb\\formie\\fields\\formfields\\SingleLineText::isEmpty()",
  "code": 0,
  "error": "Calling unknown method: verbb\\formie\\fields\\formfields\\SingleLineText::isEmpty()",
  "exception": "yii\\base\\UnknownMethodException",
  "file": "/***/vendor/yiisoft/yii2/base/Component.php",
  "line": 300,
  "trace": [
    {
      "file": "/***/vendor/verbb/formie/src/elements/Submission.php",
      "line": 784,
      "function": "__call",
      "class": "yii\\base\\Component",
      "type": "->"
    },
    {
      "file": "/***/vendor/verbb/formie/src/controllers/SubmissionsController.php",
      "line": 1116,
      "function": "setFieldValuesFromRequest",
      "class": "verbb\\formie\\elements\\Submission",
      "type": "->"
    },
    {
      "file": "/***/vendor/verbb/formie/src/controllers/SubmissionsController.php",
      "line": 463,
      "function": "_populateSubmission",
      "class": "verbb\\formie\\controllers\\SubmissionsController",
      "type": "->"
    },
    {
      "function": "actionSubmit",
      "class": "verbb\\formie\\controllers\\SubmissionsController",
      "type": "->"
    },
    {
      "file": "/***/vendor/yiisoft/yii2/base/InlineAction.php",
      "line": 57,
      "function": "call_user_func_array"
    },
    {
      "file": "/***/vendor/yiisoft/yii2/base/Controller.php",
      "line": 178,
      "function": "runWithParams",
      "class": "yii\\base\\InlineAction",
      "type": "->"
    },
    {
      "file": "/***/vendor/yiisoft/yii2/base/Module.php",
      "line": 552,
      "function": "runAction",
      "class": "yii\\base\\Controller",
      "type": "->"
    },
    {
      "file": "/***/vendor/craftcms/cms/src/web/Application.php",
      "line": 301,
      "function": "runAction",
      "class": "yii\\base\\Module",
      "type": "->"
    },
    {
      "file": "/***/vendor/craftcms/cms/src/web/Application.php",
      "line": 625,
      "function": "runAction",
      "class": "craft\\web\\Application",
      "type": "->"
    },
    {
      "file": "/***/vendor/craftcms/cms/src/web/Application.php",
      "line": 280,
      "function": "_processActionRequest",
      "class": "craft\\web\\Application",
      "type": "->"
    },
    {
      "file": "/***/vendor/yiisoft/yii2/base/Application.php",
      "line": 384,
      "function": "handleRequest",
      "class": "craft\\web\\Application",
      "type": "->"
    },
    {
      "file": "/***/public_html/index.php",
      "line": 26,
      "function": "run",
      "class": "yii\\base\\Application",
      "type": "->"
    }
  ]
}
``` 

### Steps to reproduce

1. Create a new form and use the default Contact form
2. Add a `Single Line Text` field
3. In the field set `Visibility` to `Disabled`
4. Save the form
5. Go to the form and fill out the fields
6. Submit the form
7. See the error
8. Check the browser console for the Network tab
9. Inspect the POST request just made and in the Response see the error and stacktrace as shown above

### Form settings

- Multi-page form: Yes and No (happens on both)
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

4.3.7.1

### Plugin version

2.0.21

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.21",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.22](https://github.com/verbb/formie/releases/tag/2.0.22)
